src/components/join-flow/email-step.jsx

```diff
@@ -13,9 +16,18 @@ class EmailStep extends React.Component {
         super(props);
         bindAll(this, [
             'handleValidSubmit',
+            'validateEmailIfPresent',
             'validateForm'
         ]);
     }
+    validateEmailIfPresent (email) {
+        if (!email) return null; // skip validation if email is blank; null indicates valid
+        const localResult = isEmail.validate(email);
+        if (localResult) {
+            return null; // TODO: validate email address remotely
```

Comment:
Maybe you should check `.gov.xx`, `.go.xx` or `.gov` because no governments want spam mail & there will be no use